src/convertToObject.js

```diff
@@ -3,16 +3,25 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const css = {};
+
+  sourceString.split(';')
+    .filter((el) => el.trim())
```

Comment:
Point 4 from checklist.
```suggestion
  sourceString
    .split(';')
    .filter((el) => el.trim())
```